packages/favorites-dialog/src/FavoritesDialog.js

```diff
@@ -51,14 +53,43 @@ class FavoritesDialog extends Component {
             </Provider>
         );
     }
+
+    render() {
+        if (this.props.insertTheme) {
+            return (
+                <MuiThemeProvider
+                    theme={createMuiTheme(
+                        // override the lg width to make it wider
+                        Object.assign({}, mui3theme, {
+                            overrides: {
```

Comment:
I'm concerned that doing it like this will delete all the other overrides (like DialogTitle, DialogContent, DialogActions, MenuItem, Select). One way to avoid this is by using immutability-helper: https://github.com/kolodny/immutability-helper (look at the `update` function)

Reply:
Fixed by cloning the `mui3theme` object and overriding only the `paperWidthLg` key.